plugins/inputs/uwsgi/README.md

```diff
@@ -0,0 +1,94 @@
+# uWSGI
+
+The uWSGI gathers metrics about uWSGI using its [Stats Server][stats_server].
+
+### Configuration
+
+```toml
+[[inputs.uwsgi]]
+    ## List with urls of uWSGI Stats servers. Url must match pattern:
+    ## scheme://address[:port]
+    ##
+    ## For example:
+    ## servers = ["tcp://localhost:5050", "http://localhost:1717", "unix:///tmp/statsock"]
+    servers = []
+    ## General connection timout in seconds
+    timeout = 5
+```
+
+### Measurements and fields
+
+- uwsgi_overview
+    - listen_queue
+    - listen_queue_errors
+    - load
+    - signal_queue
+
+- uwsgi_workers
+    - requests
+    - accepting
+    - delta_request
+    - exceptions
+    - harakiri_count
+    - signals
+    - signal_queue
+    - status
+    - rss
+    - vsz
+    - running_time
+    - last_spawn
+    - respawn_count
+    - tx
+    - avg_rt
+
+- uwsgi_apps
+    - modifier1
+    - requests
+    - startup_time
+    - exceptions
+
+- uwsgi_cores
+    - requests
+    - static_requests
+    - routed_requests
+    - offloaded_requests
+    - write_errors
+    - read_errors
+    - in_request
+
+###  Tags
+
+- uwsgi_overview
+    - url
+    - pid
+    - uid
+    - gid
+    - version
+    - cwd
```

Comment:
Maybe `cwd` should be a field or not collected, as it doesn't seem very useful to me?